scylla/src/client/session.rs

```diff
@@ -806,6 +806,26 @@ impl Session {
         }
 
         let (tablet_sender, tablet_receiver) = tokio::sync::mpsc::channel(TABLET_CHANNEL_SIZE);
+
+        #[allow(unused_labels)] // Triggers when `cloud` feature is disabled.
+        let address_translator = 'translator: {
+            #[cfg(feature = "cloud")]
+            if let Some(translator) = config.cloud_config.clone() {
+                if config.address_translator.is_some() {
+                    // This can only happen if the user builds SessionConfig by hand, as SessionBuilder in cloud mode prevents setting custom AddressTranslator.
+                    warn!(
+                        "Overriding user-provided AddressTranslator with Scylla Cloud AddressTranslator due \
+                            to CloudConfig being provided. This is certainly an API misuse - Cloud \
+                            may not be combined with user's own AddressTranslator."
+                    )
+                }
+
+                break 'translator Some(translator as Arc<dyn AddressTranslator>);
+            }
+
+            config.address_translator
+        };
```

Comment:
Please use `if else` instead of break with label and value.

Reply:
I wrote it this way because it appeared to me to be the most readable way. If you have a better way on your mind, please write it here.